package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh

```diff
@@ -152,24 +152,37 @@ proto_qmi_setup() {
 		esac
 	fi
 
-	[ -n "$plmn" ] && {
-		local mcc mnc
-		if [ "$plmn" = 0 ]; then
+	json_load "$(uqmi -s -d "$device" --get-plmn)"
```

Comment:
Shouldn't there be `json_init` above the `json_load`?

Reply:
No, I don't think so. There is also no json_init [here](https://github.com/openwrt/openwrt/blob/master/package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh#L108), [here](https://github.com/openwrt/openwrt/blob/master/package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh#L320), [here](https://github.com/openwrt/openwrt/blob/master/package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh#L361) or [here](https://github.com/openwrt/openwrt/blob/master/package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh#L427) above the json_load in that file.

Reply:
json_init seems only necessary if you build an json object from scratch.